app.py

```diff
@@ -1,13 +1,59 @@
-from flask import Flask
+from flask import Flask, jsonify
+import requests
+import os
 
 
 app = Flask(__name__)
 
+base_URL = os.environ.get('BASE_URL',
+                          'https://cplab1.techservices.illinois.edu:443/api')
+
 
 # Canary
```

Comment:
Can you define the purpose of the Canary here? Or in the README if it is too long?

Reply:
This could be the docstring for the function